skaffold-operator.yaml

```
@@ -4,7 +4,7 @@ metadata:
   name: gardener-operator
 build:
   artifacts:
-    - image: eu.gcr.io/gardener-project/gardener/operator
+    - image: europe-docker.pkg.dev/gardener-project/snapshots/gardener/operator
```

```suggestion
    - image: europe-docker.pkg.dev/gardener-project/releases/gardener/operator
```
please see [this comment](https://github.com/gardener/gardener/pull/8968#issuecomment-1863415635), also for the rest of this file.